### PR TITLE
Add config item enable disable

### PR DESCRIPTION
**Pull request by bmhughes**
# Description

Add enable/disable actions to control a configuration item whilst leaving the configuration file in place.

## Issues Resolved

None

## Check List

- [x] All tests pass. See TESTING.md for details.
- [x] New functionality includes testing.
- [x] New functionality has been documented in the README if applicable.

### CODE REVIEW

**Review by Xorima — APPROVED** [review]

File: test/integration/config/source_test.rb [hunk 1]
```diff
@@ -16,6 +16,15 @@
   end
 end
 
+%w(s_test_network_multiple_disabled).each do |file|
+  describe file("/etc/syslog-ng/source.d/#{file}.conf.disabled") do
+    it { should exist }
+    its('type') { should cmp 'file' }
+    it { should be_file }
```
Comment (Xorima) [inline test/integration/config/source_test.rb:23]: duplication?

Comment (bmhughes) [reply test/integration/config/source_test.rb:23]: I'd say so, copy paste special I imagine.

File: test/integration/default/default_test.rb [hunk 2]
```diff
@@ -19,7 +19,7 @@
 
 describe file('/etc/syslog-ng/syslog-ng.conf') do
   it { should exist }
-  its('type') { should cmp 'file' }
+
```
Comment (Xorima) [inline test/integration/default/default_test.rb:22]:
```suggestion
```

**Review by Xorima — APPROVED** [review]

**Comment by bmhughes** [comment]
I'm going to try and get the service changes into this from nginx as well as it uses the same pattern, it seems fine just want to test it myself first.

**Comment by Xorima** [comment]
I would rather go with another from a review perspective :+1:

**Comment by kitchen-porter** [comment]
Released as: [1.1.0](https://github.com/sous-chefs/syslog_ng/releases/tag/1.1.0)

**Pull request merged**